cymetric/metrics.py

```diff
@@ -191,6 +191,27 @@ def decay_heat(series):
 del _dhdeps, _dhschema
 
 
+# Agent Commissioning
+_csdeps = [('AgentEntry', ('SimId', 'EnterTime'), 'Prototype')]
+
+_csschema = [
+    ('SimId', ts.UUID), ('EnterTime', ts.INT), ('Prototype', ts.STRING), 
+    ('Count', ts.INT)
+    ]
+
+@metric(name='CommissioningSeries', depends=_csdeps, schema=_csschema)
```

Comment:
In Cyclus (kernel repo anyway) we have so far used the convention `Build <--> Decommission`.  That would be my preference here - just to maintain consistency.